pages/apim/3.x/installation-guide/upgrades/3.20.0/README.adoc

```diff
@@ -0,0 +1,17 @@
+= Upgrade to 3.20.0
+
+== Breaking changes
+
+=== Alert Engine
+Starting with 3.20.0, APIM is shipped with the new 2.0.0 version of the AlertEngine connector.
+
+Events are sent over http as default behaviour. In order to switch back to websocket:
+
+```yaml
+alerts:
+  alert-engine:
+    ws:
+      sendEventsOnHttp: false
+```
+
+Please refer to the AlertEngine documentation for more information: https://docs.gravitee.io/ae/apim_installation.html#event_sending_mode.
```

Comment:
```suggestion
Please see to the link:/ae/apim_installation.html#event_sending_mode[AlertEngine documentation] for more information.
```

Reply:
In theory, I would agree, but because this README is included into another page and because of this bug https://github.com/asciidoctor/jekyll-asciidoc/issues/166, I'm not sure the "link:" syntax will work

Reply:
In this case it does - I made the change locally and built it - but you are right to be cautious.

Reply:
I did the change 😉